src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariable2Two.java

```diff
@@ -49,6 +49,15 @@ class InnerClass
 interface Inter2
 {
     void method(int aParam);
+
```

Comment:
please add in Input code with catch (single and multicatch) in Interface, as we discussed above it is not very obvious.

Reply:
all changes in behavior that we see in dff report and not in out tests - is signal to extend out Input base with more diverse code.